packages/grafana-ui/src/components/Tabs/TabsBar.tsx

```diff
@@ -13,32 +13,30 @@ export interface Props {
   hideBorder?: boolean;
 }
 
-const getTabsBarStyles = stylesFactory((theme: GrafanaTheme2, hideBorder = false) => {
-  return {
-    tabsWrapper:
-      !hideBorder &&
-      css`
-        border-bottom: 1px solid ${theme.colors.border.weak};
-      `,
-    tabs: css`
-      position: relative;
-      display: flex;
-      height: ${theme.components.menuTabs.height}px;
-    `,
-  };
-});
-
-export const TabsBar = React.forwardRef<HTMLDivElement, Props>(({ children, className, hideBorder }, ref) => {
-  const theme = useTheme2();
-  const tabsStyles = getTabsBarStyles(theme, hideBorder);
+export const TabsBar = React.forwardRef<HTMLDivElement, Props>(({ children, className, hideBorder = false }, ref) => {
+  const styles = useStyles2(getStyles);
 
   return (
-    <div className={cx(tabsStyles.tabsWrapper, className)} ref={ref}>
-      <div className={tabsStyles.tabs} role="tablist">
+    <div className={cx({ [styles.tabsBorderBottom]: !hideBorder }, styles.tabsScroll, className)} ref={ref}>
```

Comment:
```suggestion
    <div className={cx(styles.tabsWrapper, hideBorder && styles.noBorder, className)} ref={ref}>
```

tabsWrapper has the scroll

Reply:
you mean to have a tabsWrapper which is
tabsWrapper: css `
border-bottom: 1px solid ${theme.colors.border.weak};
  overflow-x: auto;
`

and then another property noBorder:
noBorder: css `
border-bottom: 0;
`
?

Reply:
yes

Reply:
done. I need to check a test that is failing though